sdk/src/main/java/com/hedera/hashgraph/sdk/Client.java

```diff
@@ -1240,6 +1242,34 @@ public Client setCloseTimeout(Duration closeTimeout) {
         return this;
     }
 
+    /**
+     * Maximum amount of time a gRPC request can run
+     *
+     * @return                          the gRPC deadline value
+     */
+    @SuppressFBWarnings(
+        value = "EI_EXPOSE_REP",
+        justification = "A Duration can't actually be mutated"
+    )
+    public synchronized Duration getGrpcDeadline() {
```

Comment:
Remove synchronized on get as it will be a performance bottleneck. Can also remove it on set if you use volatile or AtomicReference for the field

Reply:
Makes sense. However getRequestTimeout() is also synchronized, is that on purpose?

Reply:
I switched to AtomicRef for the grpcDeadline. Probably in a separate work item we'll need to refactor the rest of the Client's members for consistency...